### ISSUE TITLE
Use consistent language for setting a field value

### ISSUE THREAD

**Issue by odscjen** [open-contracting/european-union-support#151]
There are 2 general ways of setting an OCDS field value. In the first the value of the BT is mapped to the OCDS field. In the second the OCDS field is set to a fixed value defined in the guidance.

For example the first of those patterns can be seen in:

BT-10-Procedure-Buyer
> - [Get the organization for the buyer](common-operations.md#get-the-organization-for-the-buyer) and add a `Classification` object to its `.details.classifications` array.
> - **Map the value of this field to the classification's `.id`.**

and 

BT-123-Lot
> [Get the lot for the ProcurementProjectLot](common-operations.md#get-the-lot-for-a-procurementprojectlot) and **map to its `.techniques.electronicAuction.url`**.

and the second pattern can be seen in:

BT-119-LotResult
> If `true`, **set the lot's `.techniques.dynamicPurchasingSystem.status` to 'terminated'**. Otherwise, do nothing.

But there are other examples which don't quite fit to either pattern. For example:

BT-13713-LotResult
> If there is an `Award` in `awards` whose `.id` is equal to the value of `ancestor::efac:LotResult/cbcID`, discard. Otherwise:
>      - Add an `Award` to `awards`
        - **Set its `.id` to the value of `ancestor::efac:LotResult/cbc:ID`**

where we've used the command "set" but are referencing a specific value within the XML.

I feel we should have consistent language wherever possible within the guidance so am proposing we follow 2 simple patterns for OCDS fields that aren't arrays:
1. If the value is the value of a field, either the BT in question or another field defined using its XPATH then use the command "map".
2. If the value is a fixed value defined in the guidance use the command "set".

**Comment by odscjen** [comment]
Do we also want to consistently use the phrasing "the value of this field" to make it 100% clear what it is that's being mapped?

**Comment by duncandewhurst** [comment]
> 1. If the value is the value of a field, either the BT in question or another field defined using its XPATH then use the command "map".
> 2. If the value is a fixed value defined in the guidance use the command "set".

Sounds good! To be clear that would change the guidance for BT-13713-LotResult to:

> If there is an `Award` in `awards` whose `.id` is equal to the value of `ancestor::efac:LotResult/cbcID`, discard. Otherwise:
> - Add an `Award` to `awards`
> - **Map `ancestor::efac:LotResult/cbc:ID` to its `.id`**

For the sake of brevity, I'm fine with using 'map to' for straightforward mappings and only using 'map the value of this field to' when there might be ambiguity over what is being mapped.

**Comment by odscjen** [comment]
great, I'm also fine with only using 'map the value of this field' when there's possible ambiguity :)

**Comment by duncandewhurst** [comment]
Suggest this something that is a nice to have, i.e. doesn't need to be done before the profile is shared.

**Comment by jpmckinney** [comment]
Yes, agreed on nice-to-have.

**Comment by jpmckinney** [comment]
Noting that we should also use consistent punctuation as part of this, e.g. most of the time, the verb ("set" or "add") should have a comma before the conjunction (e.g. "Get the .... , and set ...").

**Comment by odscjen** [comment]
note to future me, got to BT-743

**Comment by odscjen** [comment]
completed
